source/documentation/other-topics/migrate-to-live.html.md.erb

```diff
@@ -421,6 +421,17 @@ To roll back, going through the process in reverse, depending on how far you got
 - (Step 1 - Agree a time to perform the migration -
   You can let your team know that they are no longer restricted from changing the environment config)
 
+## Frequently asked questions
+
+1. **How to ensure the alerts are migrated?**
+
+    When you migrate your namespace to `live` cluster in [Step 3](#step-3-migrate-your-namespace-environment-to-quot-live-quot), 
+    the namespace files related to monitoring and alerting such as ServiceMonitors, NetworkPolicy and prometheusRule in [the environments repo][env-repo]
+    are copied and applied to the `live` cluster. 
+    
+    After you deploy your application to the `live` cluster, in the [Cloud Platform Prometheus](https://prometheus.live.cloud-platform.service.justice.gov.uk/) targets, Rules and Service Discovery pages, ensure your application is listed.
+    In the [Cloud Platform Alertmanager](https://alertmanager.live.cloud-platform.service.justice.gov.uk) -> Status, ensure your application severity,receiver and Channel information is added to the list.
```

Comment:
Need to add a space here: `severity,receiver`.

Also, is it worth specifying how a user can `kubectl get prometheusrules` etc.